whitelist_python.py

```diff
@@ -0,0 +1,3 @@
+import collections
+
+collections.defaultdict(list).default_factory
```

Comment:
OK, as said above, W should be

```
import collections

collections.defaultdict(list).default_factory = None
collections.defaultdict(list).default_factory
```

Then we need the following tests:

python(W) -> correct code
vulture(W) -> no dead code